src/components/ContentNode/Table.vue

```diff
@@ -35,6 +47,14 @@ export default { name: 'Table' };
   -webkit-overflow-scrolling: touch;
 }
 
+table {
+  border-style: hidden;
+
+  &.spanned {
```

Comment:
CC @SamLanier to explain the reasoning.

Reply:
@mportiz08 The vertical lines help with parsing more technical data (vs. marketing tables). In this example you showed, you wouldn't need to use any table spanning, you would just have a simple table with empty entries and the table would look right.

Reply:
I was more curious about the difference in width. Previously tables were only as big as needed based on the content and with this change the tables would always be as wide as they could be, regardless of the content, which may work well for some content but not for others.

And more importantly, this change (as-is) would _only_ take place for tables that have cells that span more than a single cell, and not for normal tables, which seems kind of odd to me.

And just to be clear, the first screenshot was without this PR. With this PR and without the table width specific change, this is what that example would look like:
<img width="1436" alt="Screenshot 2022-09-16 at 10 50 16 AM" src="https://user-images.githubusercontent.com/212918/190701754-7d85bdee-e993-4ad2-93c7-d38771a55610.png">

That is more aligned with what I was expecting, as it could still easily adapt the width to fit the given content while still maintaining the span data.

Reply:
I think the reasoning was to match Github's table rendering (which I may have goofed). 

If you want a cell to be 2x wide as the rest, you can `colspan` it with an empty cell and it becomes 2x. 

In our current existing design that will not happen. But by adding `width: 100%; table-layout: fixed;` to the table, it baheves just like that. Yet that is a pretty big breaking change with current tables, so I advised if we are doing this, to only do it to spanned tables.

Reply:
> In our current existing design that will not happen.

But it will—just like in the screenshot from my most recent comment above. I think that is where my main confusion is from. I may be misunderstanding the issue, sorry!

Reply:
We discussed this in person and agreed that it would be best to leave this out for now as something that we can either look at wholesale applying to the default table styles or exploring as something that can be an opt-in by using some syntax feature.